src/rio.c

```diff
@@ -46,6 +46,7 @@
 
 
 #include "fmacros.h"
+#include <assert.h>
```

Comment:
let's switch to this, so we get a better crash report if it ever happens.
```suggestion
#include "redisassert.h"
```